### PR TITLE
Added global stylings and necessary components

### PR DESCRIPTION
**Pull request by tylerpierson**
Added in some global stylings and added in some components that we'll be needing later on. Threw a button and input on the HomePage just for people to checkout to see if they have any recommendations. Also had to comment out some backend code that was throwing errors. If that is messing up the merge, feel free to take the comments out.

### CODE REVIEW

File: src/App.module.scss [hunk 1]
```diff
@@ -1,25 +1,35 @@
...
+
+button {
+    background-color: var(--btn-color);
+    border-top-left-radius: 1rem;
+    border-bottom-right-radius: 1rem;
+}
+
+button:hover {
+    cursor: pointer;
+}
```
Comment (dwheeler7) [inline src/App.module.scss:35]: Why hard code in buttons styles instead of using button component?

File: src/App.module.scss [hunk 2]
```diff
@@ -1,25 +1,35 @@
...
+    margin-left: 1rem;
+    font-size: 2rem;
+    padding: 1rem;
+    color: var(--input-color);
 }
+
+input:focus {
+    outline: none;
+    box-shadow: 0 0 1rem var(--btn-color);
+}
```
Comment (dwheeler7) [inline src/App.module.scss:20]: Why not use input component?

File: src/App.module.scss [hunk 3]
```diff
@@ -1,25 +1,35 @@
...
+
+label {
+    font-size: 2.3rem;
+    padding: 1rem;
+}
```
Comment (dwheeler7) [inline src/App.module.scss:25]: Same question

**Review by dwheeler7 — CHANGES_REQUESTED** [review]

File: src/App.module.scss (hunk 1 above)
Comment (tylerpierson) [reply src/App.module.scss:35]: I am using a button component. Just typed in the wrong styling sheet.

File: src/App.module.scss (hunk 2 above)
Comment (tylerpierson) [reply src/App.module.scss:20]: Wasn't necessary for my example input. Once I've got consensus, I can implement one.

File: src/App.module.scss (hunk 3 above)
Comment (tylerpierson) [reply src/App.module.scss:25]: Same response

**Review by dwheeler7 — APPROVED** [review]

**Pull request merged**